### PR TITLE
[Docs][Connector-V2][Mysql] Refactor connector-v2 docs using unified format Mysql

### PR DESCRIPTION
**Pull request by zhilinli123**
subtask of https://github.com/apache/incubator-seatunnel/issues/4572

### CODE REVIEW

**Comment by zhilinli123** [comment]
Let's see if we have any questions thanks`

File: docs/en/connector-v2/sink/Mysql.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
+# JDBC
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Mysql.md]: Update `JDBC` to `MySQL` ?

File: docs/en/connector-v2/sink/Mysql.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+| password                                  | String  | No       | -             | Connection instance password                                                                                                                                                                                                                 |
+| query                                     | String  | No       | -             | Use this sql write upstream input datas to database. e.g `INSERT ...`                                                                                                                                                                        |
+| database                                  | String  | No       | -             | Use this `database` and `table-name` auto-generate sql and receive upstream input datas write to database.<br/>This option is mutually exclusive with `query` and has a higher priority.                                                     |
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Mysql.md]: `query` have the higher priority.

File: docs/en/connector-v2/sink/Mysql.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+| table                                     | String  | No       | -             | Use database and this table-name auto-generate sql and receive upstream input datas write to database.<br/>This option is mutually exclusive with query and has a higher priority.                                                           |
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Mysql.md]:
```suggestion
| table                                     | String  | No       | -             | Use database and this table-name auto-generate sql and receive upstream input datas write to database.<br/>This option is mutually exclusive with `query` and has a higher priority.                                                           |
```

File: docs/en/connector-v2/sink/Mysql.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+## tips
+
+> If partition_column is not set, it will run in single concurrency, and if partition_column is set, it will be executed  in parallel according to the concurrency of tasks.
+
+## Task Example
+
+### simple:
+
+> This is a simple example of Mysql via Jdbc Sink
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Mysql.md]: Please add more information about this example. For example:

**This example defines a SeaTunnel synchronization task that automatically generates data through FakeSource and sends it to JDBC Sink. FakeSource generates a total of 16 rows of data (`row.num=16`), with each row having two fields, name (string type) and age (int type). The final target table is `test_table`  will also be 16 rows of data in the table.**

> Before run this job, you need create database `test` and table `test_table` in your mysql. And if you have not yet installed and deployed SeaTunnel, you need to follow the instructions in [Install SeaTunnel](../../start-v2/locally/deployment.md) to install and deploy SeaTunnel. And then follow the instructions in [Quick Start With SeaTunnel Engine](../../start-v2/locally/quick-start-seatunnel-engine.md) to run this job.

File: docs/en/connector-v2/sink/Mysql.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+
+    is_exactly_once = "true"
+
+    xa_data_source_class_name = "com.mysql.cj.jdbc.MysqlXADataSource"
+}
+```
+
+### CDC(Change data capture) event
+
+> CDC change data is also supported by us
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Mysql.md]: Some examples we need to tell the user which options is needed. For example:

**In this case, you need config `database`, `table` and `primary_keys`.**

File: docs/en/connector-v2/source/Mysql.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
+# JDBC
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/Mysql.md]:
```suggestion
# MySQL
```

File: docs/en/connector-v2/source/Mysql.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+| query                        | String | Yes      | -               | Query statement                                                                                                                                                                                                                                                   |
+| connection_check_timeout_sec | Int    | No       | 30              | The time in seconds to wait for the database operation used to validate the connection to complete                                                                                                                                                                |
+| partition_column             | String | No       | -               | The column name for parallelism's partition, only support numeric type.                                                                                                                                                                                           |
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/Mysql.md]: Only support numeric type primary key, and only can config one column.

File: docs/en/connector-v2/source/Mysql.md [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+
+## tips
+
+> If partition_column is not set, it will run in single concurrency, and if partition_column is set, it will be executed  in parallel according to the concurrency of tasks.
+
+## Task Example
+
+### simple:
+
+> This is to access your data source according to your query parameter We can use this if we don't have a speed requirement
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/Mysql.md]: Same as above, we need let user know what does this example do.

Comment (zhilinli123) [reply docs/en/connector-v2/source/Mysql.md]: Done

File: docs/en/connector-v2/sink/Mysql.md [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+      }
+    }
+  }
+  # If you would like to get more information about how to configure seatunnel and see full list of source plugins,
+  # please go to https://seatunnel.apache.org/docs/category/source-v2
+}
+
+transform {
+  # If you would like to get more information about how to configure seatunnel and see full list of transform plugins,
+  # please go to https://seatunnel.apache.org/docs/category/transform
```
Comment (TaoZex) [inline docs/en/connector-v2/sink/Mysql.md]:
```suggestion
  # please go to https://seatunnel.apache.org/docs/category/transform-v2
```
image

The transform has been removed, use transform-v2

Comment (zhilinli123) [reply docs/en/connector-v2/sink/Mysql.md]: done

Comment (TaoZex) [reply docs/en/connector-v2/sink/Mysql.md]: LGTM. Please wait for CI.

Comment (zhilinli123) [reply docs/en/connector-v2/sink/Mysql.md]: @TyrantLucifer @hailin0 @EricJoy2048 @TaoZex Please review

**Review by EricJoy2048 — DISMISSED** [review]
LGTM

**Review by TaoZex — DISMISSED** [review]
LGTM

File: docs/en/connector-v2/sink/Mysql.md [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+## Description
+
+Write data through jdbc. Support Batch mode and Streaming mode, support concurrent writing, support exactly-once
+semantics (using XA transaction guarantee).
+
+## Supported DataSource list
+
+| datasource |              supported versions               |          driver          |                  url                  |                                   maven                                   |
+|------------|-----------------------------------------------|--------------------------|---------------------------------------|---------------------------------------------------------------------------|
+| mysql      | Depending on your Database dependency version | com.mysql.cj.jdbc.Driver | jdbc:mysql://localhost:3306:3306/test | [Download](https://mvnrepository.com/artifact/mysql/mysql-connector-java) |
```
Comment (FlechazoW) [inline docs/en/connector-v2/sink/Mysql.md]: Different dependency version has different driver class.

Comment (zhilinli123) [reply docs/en/connector-v2/sink/Mysql.md]: Done

**Review by EricJoy2048 — DISMISSED** [review]
LGTM.

This is a good job. Thanks for you contribution.

File: docs/en/connector-v2/sink/Mysql.md [hunk 11]
```diff
@@ -0,0 +1,171 @@
...
+## Support those engines
+
+> Spark<br/>
+> Flink<br/>
+> Seatunnel Zeta<br/>
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [X] [stream](../../concept/connector-v2-features.md)
```
Comment (hailin0) [inline docs/en/connector-v2/sink/Mysql.md]: remove `stream `

**Review by hailin0 — CHANGES_REQUESTED** [review]

File: docs/en/connector-v2/sink/Mysql.md (hunk 11 above)
Comment (EricJoy2048) [reply docs/en/connector-v2/sink/Mysql.md]: Yes, Sink connectors do not differentiate between streams and batch.

Comment (zhilinli123) [reply docs/en/connector-v2/sink/Mysql.md]: Done Run Ci  thanks

File: docs/en/connector-v2/source/Mysql.md [hunk 12]
```diff
@@ -0,0 +1,153 @@
...
+## Support those engines
+
+> Spark<br/>
+> Flink<br/>
+> Seatunnel Zeta<br/>
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [ ] [stream](../../concept/connector-v2-features.md)
```
Comment (hailin0) [inline docs/en/connector-v2/source/Mysql.md:14]:
```suggestion
```

Comment (zhilinli123) [reply docs/en/connector-v2/source/Mysql.md:14]: Please review

Comment (zhilinli123) [reply docs/en/connector-v2/source/Mysql.md:14]: @hailin0 Please review

File: docs/en/connector-v2/sink/Mysql.md [hunk 13]
```diff
@@ -0,0 +1,170 @@
...
+
+    max_retries = 0
+    user = "root"
+    password = "123456"
+    query = "insert into test_table(name,age) values(?,?)"
+
+    is_exactly_once = "true"
+
+    xa_data_source_class_name = "com.mysql.cj.jdbc.MysqlXADataSource"
+}
```
Comment (hailin0) [inline docs/en/connector-v2/sink/Mysql.md]:
```suggestion
sink {
    jdbc {
        url = "jdbc:mysql://localhost:3306/test"
        driver = "com.mysql.cj.jdbc.Driver"
    
        max_retries = 0
        user = "root"
        password = "123456"
        query = "insert into test_table(name,age) values(?,?)"
    
        is_exactly_once = "true"
    
        xa_data_source_class_name = "com.mysql.cj.jdbc.MysqlXADataSource"
    }
}
```

File: docs/en/connector-v2/sink/Mysql.md [hunk 14]
```diff
@@ -0,0 +1,170 @@
...
+    url = "jdbc:mysql://localhost:3306/test"
+    driver = "com.mysql.cj.jdbc.Driver"
+    user = "root"
+    password = "123456"
+    
+    # You need to configure both database and table
+    database = test
+    table = sink_table
+    primary_keys = ["id","name"]
+}
```
Comment (hailin0) [inline docs/en/connector-v2/sink/Mysql.md]:
```suggestion
sink {
    jdbc {
        url = "jdbc:mysql://localhost:3306/test"
        driver = "com.mysql.cj.jdbc.Driver"
        user = "root"
        password = "123456"
        
        # You need to configure both database and table
        database = test
        table = sink_table
        primary_keys = ["id","name"]
    }
}
```

File: docs/en/connector-v2/source/Mysql.md (hunk 12 above)
Comment (hailin0) [inline docs/en/connector-v2/source/Mysql.md:14]: remove `stream`

File: docs/en/connector-v2/source/Mysql.md [hunk 15]
```diff
@@ -0,0 +1,153 @@
...
+    }
+}
+
+transform {
+    # If you would like to get more information about how to configure seatunnel and see full list of transform plugins,
+    # please go to https://seatunnel.apache.org/docs/transform/sql
+}
+
+sink {
+Console {}
```
Comment (hailin0) [inline docs/en/connector-v2/source/Mysql.md]:
```suggestion
    Console {}
```

File: docs/en/connector-v2/source/Mysql.md [hunk 16]
```diff
@@ -0,0 +1,153 @@
...
+    connection_check_timeout_sec = 100
+    user = "root"
+    password = "123456"
+    # Define query logic as required
+    query = "select * from type_bin"
+    # Parallel sharding reads fields
+    partition_column = "id"
+    # Number of fragments
+    partition_num = 10
+}
```
Comment (hailin0) [inline docs/en/connector-v2/source/Mysql.md]:
```suggestion
source {
    Jdbc {
        url = "jdbc:mysql://localhost:3306/test?serverTimezone=GMT%2b8"
        driver = "com.mysql.cj.jdbc.Driver"
        connection_check_timeout_sec = 100
        user = "root"
        password = "123456"
        # Define query logic as required
        query = "select * from type_bin"
        # Parallel sharding reads fields
        partition_column = "id"
        # Number of fragments
        partition_num = 10
    }
}
```

File: docs/en/connector-v2/source/Mysql.md [hunk 17]
```diff
@@ -0,0 +1,153 @@
...
+    password = "123456"
+    # Define query logic as required
+    query = "select * from type_bin"
+    partition_column = "id"
+    # Read start boundary
+    partition_lower_bound = 1
+    # Read end boundary
+    partition_upper_bound = 500
+    partition_num = 10
+}
```
Comment (hailin0) [inline docs/en/connector-v2/source/Mysql.md]:
```suggestion
source {
    Jdbc {
        url = "jdbc:mysql://localhost:3306/test?serverTimezone=GMT%2b8"
        driver = "com.mysql.cj.jdbc.Driver"
        connection_check_timeout_sec = 100
        user = "root"
        password = "123456"
        # Define query logic as required
        query = "select * from type_bin"
        partition_column = "id"
        # Read start boundary
        partition_lower_bound = 1
        # Read end boundary
        partition_upper_bound = 500
        partition_num = 10
    }
}
```

Comment (zhilinli123) [reply docs/en/connector-v2/source/Mysql.md]: Thank you for your careful review. I have corrected it
@hailin0

**Comment by TyrantLucifer** [comment]
waiting CI/CD

**Review by EricJoy2048 — APPROVED** [review]

**Review by TaoZex — APPROVED** [review]
+1

**Review by hailin0 — APPROVED** [review]

**Pull request merged**